user-guide/Reference/DataMiner_Tools/Sync_Check.md

```diff
@@ -64,5 +64,75 @@ The script checks the following things:
 
   - Inconsistent folder name (different name or capitalization)
 
+  ## Examples of the Syncinfo output file
```

Comment:
Is there a specific reason why you used the numbering format "1.1", "1.2", etc. in the section below this? We don't use this anywhere on docs, so I'd prefer not to do so here if at all possible.